DQMOffline/Trigger/python/TopMonitoring_Client_cff.py

```diff
@@ -1,4 +1,174 @@
 import FWCore.ParameterSet.Config as cms
 
+topEfficiency_elejets = cms.EDAnalyzer("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/TopHLTOffline/TopMonitor/EleJet/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "effic_metME       'efficiency vs MET; MET [GeV]; efficiency' metME_numerator       metME_denominator",
```

Comment:
sorry because I'm reverse engineering these changes I think I got them wrong
cms.EDAnalyzer("DQMGenericClient",

to

 DQMEDHarvester("DQMGenericClient",

and you need to have 

from DQMServices.Core.DQMEDHarvester import DQMEDHarvester

in your config